Jenkinsfile

```diff
@@ -79,29 +79,49 @@ pipeline {
       }
     }
 
-    stage('Publish client Docker image') {
-      steps {
-        withCredentials(
-          [
-            usernamePassword(
-              credentialsId: 'conjur-jenkins-api',
-              usernameVariable: 'GIT_USER',
-              passwordVariable: 'GIT_PASSWORD'
-            )
-          ]
-        ) {
-            sh '''
-                git config --local credential.helper '! echo username=${GIT_USER}; echo password=${GIT_PASSWORD}; echo > /dev/null'
-                git fetch --tags
-                export GIT_DESCRIPTION=$(git describe --tags)
-                summon ./bin/publish
-            '''
+    stage('Release') {
+      parallel {
+        stage('Push Images') {
+          steps {
+            script {
+              BRANCH_NAME=env.BRANCH_NAME
```

Comment:
I'm not sure what this for, may as well use ${env.BRANCH_NAME} on L102?

Reply:
I need the BRANCH_NAME inside publish to determine when to push our edge tag. We want to only push when on master so I need the `BRANCH_NAME=env.BRANCH_NAME` from the script and output the result. I tried via scripts but I would also get HEAD instead of the actual branch